### DISCUSSION TITLE
Need to restrict access to database from Budibase Cloud

### DISCUSSION THREAD

**Discussion by kpturner** [Budibase/budibase#10984] [Help] [answered]
I am using Budibase in the cloud and have imported a postgreSQL schema that is running in AWS RDS.    In order to ensure that BB has access to the database I have had to open up the firewall to `0.0.0.0/0`.   However, I need to narrow this down to only the IP range that BB cloud will use rather than the entire DB.    Is it possible to provide that?

**Comment by ConorWebb96** [comment, marked as answer]
Hey @kpturner,

The whitelisting docs provides all the necessary IPs needed for Budibase. This can be found here; https://docs.budibase.com/docs/whitelisting, using these IPs you should be able to create a security group for your RDS instance that only allows our IP addresses.

I hope this helps!

**Comment by kpturner** [comment]
Yes that seems to work thanks
